Vagrantfile

```diff
@@ -59,12 +59,12 @@ Vagrant.configure(2) do |config|
       postgresql_is_local: true
     }
 
-    ansible.playbook = "figgy.yml"
+    ansible.playbook = "pas.yml"
 
     # update start_at_task and re-run `vagrant provison` if your configuration scripts fail on a particular task
     # and you want to restart the provisioning at the step where the failure occurred
     # ansible.start_at_task = "packages | update apt cache"
-    config.ssh.private_key_path = "pulsys_rsa_key"
+    config.ssh.private_key_path = "~/.ssh/pulsys_rsa_key"
```

Comment:
this file is not managed by git. No need to change the path. Look at the `.gitignore` file